<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubSSJJWidget</class>
 <widget class="QWidget" name="SubSSJJWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="toolPart">
     <item>
      <widget class="QPushButton" name="scriptPushButton">
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="toolTip">
        <string>发现</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none; 
width: 20px; height: 50px;
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icon/resources/icons/script.png</normaloff>:/icon/resources/icons/script.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="localMusicPushButton">
       <property name="toolTip">
        <string>本地音乐</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none; 
width: 20px; height: 50px;
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icon/resources/icons/spider.png</normaloff>:/icon/resources/icons/spider.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingPushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none; 
width: 20px; height: 50px;
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icon/resources/icons/setting.png</normaloff>:/icon/resources/icons/setting.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="toolTip">
        <string>关闭音乐盒</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none; 
width: 20px; height: 50px;
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icon/resources/icons/IconamoonCloseCircle1.svg</normaloff>:/icon/resources/icons/IconamoonCloseCircle1.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="settingWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="launcherGroupBox">
          <property name="title">
           <string>启动器</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="pathHorizontalLayout">
             <item>
              <widget class="QLabel" name="launcherPathLabel">
               <property name="text">
                <string>启动器路径</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="launcherPathLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="chooseLauncherPathPushButton">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    border: none; 
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resource.qrc">
                 <normaloff>:/icon/resources/icons/PhFolderNotchOpen.svg</normaloff>:/icon/resources/icons/PhFolderNotchOpen.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>18</width>
                 <height>18</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="loadSettingHorizontalLayout">
             <item>
              <widget class="QLabel" name="loadingTimeLabel">
               <property name="text">
                <string>游戏加载时间/s</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="loadingTimeLineEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="scriptWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="scriptSettingWidget">
          <item>
           <widget class="QGroupBox" name="basicInfoGroupBox">
            <property name="title">
             <string>基础信息</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="2">
              <widget class="QLabel" name="singleScriptTimeLabel">
               <property name="toolTip">
                <string>单次脚本最大执行时间（游戏内）</string>
               </property>
               <property name="text">
                <string>超时设定/分</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="moveSpeedLineEdit"/>
             </item>
             <item row="0" column="4">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="moveSpeedLabel">
               <property name="text">
                <string>移速</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="singleScriptTimeLineEdit">
               <property name="toolTip">
                <string>单次脚本最大执行时间（游戏内）</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="LDTab">
             <attribute name="title">
              <string>王者乱斗</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="2" column="2">
               <widget class="QLabel" name="LDScriptLabel">
                <property name="toolTip">
                 <string>可选</string>
                </property>
                <property name="text">
                 <string>脚本路径</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="LDRunTimesLineEdit"/>
              </item>
              <item row="2" column="3">
               <widget class="QLineEdit" name="LDScriptPathLineEdit">
                <property name="toolTip">
                 <string>可选</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="4">
               <widget class="QPushButton" name="chooseLDScriptPathPushButton">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    border: none; 
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../resource.qrc">
                  <normaloff>:/icon/resources/icons/PhFolderNotchOpen.svg</normaloff>:/icon/resources/icons/PhFolderNotchOpen.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>18</width>
                  <height>18</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="LDRunTimesLabel">
                <property name="text">
                 <string>运行次数</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QRadioButton" name="teamPropRadioButton">
                <property name="text">
                 <string>团队道具赛</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="ZXTab">
             <attribute name="title">
              <string>主线关卡</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addTaskPushButton">
            <property name="text">
             <string>加入任务列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="taskTableWidget">
            <property name="toolTip">
             <string>单击可清除任务</string>
            </property>
            <column>
             <property name="text">
              <string>任务名称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>关卡难度</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>任务次数</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>已执行次数</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>脚本</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="controlHorizontalLayout">
            <item>
             <widget class="QPushButton" name="startPushButton">
              <property name="text">
               <string>开始运行(F10)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="endPushButton">
              <property name="text">
               <string>结束运行(F11)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="testPushButton">
            <property name="text">
             <string>test</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="remindTextEdit">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
